cmd/main.go

```diff
@@ -240,7 +240,7 @@ func main() {
 	k8sComponentsRegistrationHelper := models.NewComponentsRegistrationHelper(log)
 
 	// purge the meshsync objects in the database on startup
-	models.RemoveStaleObjects(*dbHandler)
+	_ = models.RemoveStaleObjects(*dbHandler)
```

Comment:
it's better to handle the error instead of ignoring it..

Reply:
You got it. I just didn't want to step on your toes in case that ignore was intentional.